src/content/docs/build/applications/add-and-manage-applications.mdx

```diff
@@ -20,7 +20,7 @@ If you plan to manage authentication for each of your applications using Kinde,
 
 ## Add an application
 
-We recommend following [this specific procedure](/developer-tools/kinde-api/add-a-m2m-application-for-api-access/) for M2M applications that will access the Kinde Management API.
+We recommend following [this specific procedure](/developer-tools/kinde-api/about-m2m-scopes/) for M2M applications that will access the Kinde Management API.
```

Comment:
Wondering if this should be some sort of call out. Reading it in the doc it feels a bit oddly placed. as the leading paragraph before the list of steps